api/pom.xml

```diff
@@ -118,6 +118,20 @@
         </resources>
     
         <plugins>
+             <!-- Restricts the Java version to 1.8 -->
+             <plugin>
+                <groupId>org.apache.maven.plugins</groupId>
+                <artifactId>maven-compiler-plugin</artifactId>
+                <version>3.0</version>
+                <configuration>
+                    <source>1.7</source>
```

Comment:
But the comment above mentions 1.8, which is right?

Reply:
I see next commit updated comment.

Reply:
Yes indeed, but thanks a lot for your second pair of eyes! :) I'm making a lot of changes in a short period (we have a deadline today), so some extra scrutiny is highly appreciated!